### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by Nikerabbit** [wikimedia/jquery.uls#175]
I'm unable to find any information which browsers are meant to be supported by jquery.uls.

**Comment by louy2** [comment]
Since it is a jQuery plugin, I'd assume it is supported as long as jQuery is supported. Browser support of jQuery is listed here: https://jquery.com/browser-support/

**Comment by Nikerabbit** [comment]
I know we don't support IE6 or IE7. IE8 support was broken recently but then fixed. No idea about other browsers.

**Comment by louy2** [comment]
From the track record of Chrome, Firefox and Opera, I honestly don't see the point of worrying. What other less popular browsers do you need to cover?

**Comment by Nikerabbit** [comment]
I'm just looking for explicit clarity like https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix

**Comment by Nikerabbit** [comment]
Also came up in https://gerrit.wikimedia.org/r/#/c/285580/

**Comment by louy2** [comment]
> Also, we cannot add es5-shim here in mediawiki and remove the code in jquery.uls because it is used in many places outside mediawiki in various browsers-  including old browsers.

I guess what Santosh said implies a wider support than MediaWiki. I think it's safe to copy the MediaWiki support matrix to wiki of this project.

**Comment by santhoshtr** [comment]
I am inclined to use same browser policy of mediawiki(https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix) - modern browsers. At this point of time it is

![image](https://cloud.githubusercontent.com/assets/161672/15178766/36ce3900-1795-11e6-9f7c-f2713a71dccb.png)

Since we do have very limited team capacity in WMF, it would be difficult to support anything other than MediaWiki support. Also makes the codebase clean without much burden.

What do you think? @Nikerabbit ?

**Comment by Nikerabbit** [comment]
I agree that following MediaWiki makes sense for us. MediaWiki has quite wide browser support and there is no point for us making it any wider. I would be even okay with narrower support on case by case basis, i.e. we might not want to support web fonts for some old browsers in the future which are still supported by MediaWiki.

There should also be a link back to MediaWiki.org because that document gets updates and we might not always remember to update the info in uls wiki or wherever it is documented.
